docs/investigations/secure-auth-resolvers.md

```diff
@@ -0,0 +1,80 @@
+# Limit access of GraphQL objects or fields for specific resource owners
+
+### Case 1:
+
+Every object should has access to read information only about itself.
+We should restrict access to other objects of the same type, for example: runtime musn't read other runtime configuraion.
+We have two kind of such resolvers:
+* `runtime(ID)` runtime should be able to only read about itself
+* `runtimes` - this data is limited for `RuntimeAgents` with `hasScopes` directive, so it's not our case.
```

Comment:
Why this is not our case? Can you describe it a bit more?
Maybe it is worth to mention here shortly how the `hasScopes` directive works.

Reply:
`runtimes` should be called by admin or somebody with high priviledges. 
We can restrict this by `scopes`.

Reply:
```suggestion
* `runtimes` - this data is limited for `RuntimeAgents` with `hasScopes` directive, so it's not our case.
(hasScopes directive restrict access to resolver)
```